lms/static/js/spec/views/message_banner_spec.js

```diff
@@ -1,27 +1,46 @@
-define(['backbone', 'jquery', 'underscore', 'js/views/message_banner'
+define(['backbone', 'jquery', 'underscore', 'js/views/message', 'js/common_helpers/template_helpers'
        ],
-    function (Backbone, $, _, MessageBannerView) {
+    function (Backbone, $, _, MessageView, TemplateHelpers) {
         'use strict';
 
-        describe("MessageBannerView", function () {
+        describe("MessageView", function () {
 
             beforeEach(function () {
-                setFixtures('<div class="message-banner"></div>');
+                setFixtures('<div class="message-banner"></div><div class="message"></div>');
                 TemplateHelpers.installTemplate("templates/fields/message_banner");
+                TemplateHelpers.installTemplate("templates/message_view");
             });
 
-            it('renders message correctly', function() {
-                var messageSelector = '.message-banner';
-                var messageView = new MessageBannerView({
-                    el: $(messageSelector)
+            var createMessageView = function (messageContainer, templateId) {
+                return new MessageView({
+                    el: $(messageContainer),
+                    templateId: templateId
                 });
+            };
+
+            it('renders message correctly with template with no icon', function() {
```

Comment:
I would make this: "renders correctly with the /fields/message_banner template" and the one below: "renders correctly with the /message_view template".